packages/thumbprint-react/components/Modal/index.tsx

```diff
@@ -106,221 +112,189 @@ const ModalAnimatedWrapper = ({
     /**
      * Allows the page to scroll vertically at viewports larger than the small breakpoint. If
      * `false`, the modal will always be equal to or smaller than the viewport and the contents
      * of the modal will scroll, not the page itself.
      */
-    shouldPageScrollAboveSmall: PropTypes.bool,
+    shouldPageScrollAboveSmall?: boolean;
     /**
      * Should the modal appear open.
      */
-    isOpen: PropTypes.bool,
+    isOpen?: boolean;
     /**
      * Sets the max-width of the modal container.
      */
-    width: PropTypes.oneOf(['narrow', 'medium', 'wide']),
+    width?: 'narrow' | 'medium' | 'wide';
     /**
      * Sets height of the modal container above small viewport.
      * If `auto` (default), the modal height will be determined by its content.
      * Otherwise, the modal height will be fixed at some constant px.
      */
-    heightAboveSmall: PropTypes.oneOf(['auto', 'medium', 'tall']),
-};
-
-ModalAnimatedWrapper.defaultProps = {
-    children: undefined,
-    onOpenFinish: undefined,
-    onCloseFinish: undefined,
-    isOpen: false,
-    shouldCloseOnCurtainClick: true,
-    shouldPageScrollAboveSmall: true,
-    width: 'medium',
-    heightAboveSmall: 'auto',
-};
+    heightAboveSmall?: 'auto' | 'medium' | 'tall';
+}
 
-const modalHeaderPropTypes = {
+interface ModalHeaderPropTypes {
     /**
      * Content (usually a `ModalTitle` and `ModalDescription`) that appears at the top of the
      * modal.
      */
-    children: PropTypes.node.isRequired,
-};
+    children: React.ReactNode;
+}
 
-const modalTitlePropTypes = {
+interface ModalTitlePropTypes {
     /**
      * Text that describes the modal contents. It is intended for use within the `ModalHeader`.
      */
-    children: PropTypes.string.isRequired,
-};
+    children: string;
+}
 
-const modalDescriptionPropTypes = {
+interface ModalDescriptionPropTypes {
     /**
      * Text intended for use below a `ModalTitle` and within a `ModalHeader`.
      */
-    children: PropTypes.node.isRequired,
-};
+    children: React.ReactNode;
+}
 
-const modalContentPropTypes = {
+interface ModalContentPropTypes {
     /**
      * Content (usually a form) that makes up the main part of the modal.
      */
-    children: PropTypes.node.isRequired,
-};
+    children: React.ReactNode;
+}
 
-const modalContentFullBleedPropTypes = {
+interface ModalContentFullBleedPropTypes {
     /**
      * Content (usually a form) that makes up the main part of the modal.
      */
-    children: PropTypes.node.isRequired,
+    children: React.ReactNode;
     /**
      * Allows the React `className` prop to be passed through to the rendered element.
      */
-    className: PropTypes.string,
+    className?: string;
     /**
      * Allows the React `style` prop to be passed through to the rendered element.
      */
-    style: PropTypes.shape(),
-};
-
-const modalContentFullBleedDefaultProps = {
-    className: '',
-    style: {},
-};
+    style?: object;
```

Comment:
`React.CSSProperties` would be helpful here.

Reply:
done